cmd/osctl/cmd/cluster.go

```diff
@@ -289,7 +305,7 @@ func createNetwork(cli *client.Client) (types.NetworkCreateResponse, error) {
 		IPAM: &network.IPAM{
 			Config: []network.IPAMConfig{
 				{
-					Subnet: fmt.Sprintf(baseNetwork, 0) + "/24",
+					Subnet: networkCIDR,
```

Comment:
Originally thought we'd need to validate the cidr here, but I think we can actually just pass the string directly, since we ensured, and would have bailed, if the string wasn't valid on line 105 above.

Reply:
> This isn't required for the configurable docker clusters necessarily, but it felt like it'd be plenty useful. Tested on default cidr, `10.10.0.0/24`, and `10.10.0.128/25`

It is useful for running multiple talos clusters on one docker server.